lib/Rest/HttpRequest.cpp

```diff
@@ -883,7 +883,7 @@ VPackSlice HttpRequest::payload(VPackOptions const* options) {
       if (!_vpackBuilder) {
         VPackOptions validationOptions = *options;  // intentional copy
         validationOptions.validateUtf8Strings = true;
-        VPackParser parser(&validationOptions); // adding the validatin here has no effect
+        VPackParser parser(&validationOptions);
```

Comment:
I am still not getting it fully.
Does the setting of `validateUtf8Strings` to `true` here have an effect or not? Because if not, then we are effectively not validating incoming JSON data.

Reply:
It has. I think I did something wrong that that time.